docs/export_and_import_fields.rst

```diff
@@ -0,0 +1,111 @@
+rms.export_and_import_field_parameters
+=======================================
+
+The  export and import scripts desctibed below supports a workflow where
```

Comment:
I think I here will refer to _functions_, not _scripts_, since the user interface is a function. Also a typo: "desctibed"

Reply:
fixed